tests/exploreServices.spec.ts

```diff
@@ -144,9 +146,15 @@ test.describe('explore services page', () => {
         await route.fulfill({ json: {} });
       });
 
+      await page.route('**/resources/labels*', async (route) => {
+        labelsQueryCount++;
```

Comment:
Is this something we should assert on?

Reply:
Not anymore, we removed this call. Fixed